expert360/contentful.ex
# Update deps

Update deps

How to test:
- pull down the repo and checkout this branch
- in expert360_com edit /main/backend/apps/notifier/mix.exs and change the contentful dep to a path dep e.g. `{:contentful, path: "/Users/adamd/work/contentful.ex"}`
- run mix deps.get
- fire up the app (will need the contentful api keys in env)
- send an email that relies on contentful. Invite expert is an example one.
- check dev mailbox template looks correct.
